Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -71,33 +76,36 @@
  */
 public class RobotContainer
 {
-  private static RobotContainer    m_instance;
+  private static RobotContainer        m_instance;
 
   // Joysticks
-  private final XboxController     m_driverPad          = new XboxController(Constants.kDriverPadPort);
-  private final XboxController     m_operatorPad        = new XboxController(Constants.kOperatorPadPort);
+  private final XboxController         m_driverPad          = new XboxController(Constants.kDriverPadPort);
+  private final XboxController         m_operatorPad        = new XboxController(Constants.kOperatorPadPort);
 
   // The robot's shared subsystems
-  public final LED                 m_led                = new LED( );
-  public final Vision              m_vision             = new Vision( );
+  public final LED                     m_led                = new LED( );
+  public final Vision                  m_vision             = new Vision( );
 
   // These subsystems can use LED or vision and must be created afterward
-  public final Elbow               m_elbow              = new Elbow( );
-  public final Extension           m_extension          = new Extension( );
-  public final Wrist               m_wrist              = new Wrist( );
-  public final Gripper             m_gripper            = new Gripper( );
-  public final Power               m_power              = new Power( );
-  public final Swerve              m_swerve             = new Swerve( );
+  public final Elbow                   m_elbow              = new Elbow( );
+  public final Extension               m_extension          = new Extension( );
+  public final Wrist                   m_wrist              = new Wrist( );
+  public final Gripper                 m_gripper            = new Gripper( );
+  public final Power                   m_power              = new Power( );
+  public final Swerve                  m_swerve             = new Swerve( );
 
   // A chooser for autonomous commands
-  private SendableChooser<Command> m_autoChooser        = new SendableChooser<>( );
+  private SendableChooser<AutoChooser> m_autoChooser        = new SendableChooser<>( );
+  private Command                      autoCommand;
 
   // Command Scheduler
-  public Command                   m_extensionCalibrate = new ExtensionCalibrate(m_extension);
+  public Command                       m_extensionCalibrate = new ExtensionCalibrate(m_extension);
 
-  PathPlannerTrajectory            m_driveOutOfCommunityShort;
-  PathPlannerTrajectory            m_driveOutOfCommunityLong;
-  PathPlannerTrajectory            m_driveOntoChargeStation;
+  PathPlannerTrajectory                autoTrajectory;
+
+  PathPlannerTrajectory                m_driveOutOfCommunityShort;
```

Comment:
I don't think we use these three trajectory objects anymore, since we now only build one trajectory. Please confirm or delete.